applications/launchpad_v2/src/fonts.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.otf'
```

Comment:
NIT: You can move this into `src/custom.d.ts` file, so all `declare module` are kept in one place